src/cmap/connection_pool.ts

```diff
@@ -90,6 +90,8 @@ export interface ConnectionPoolOptions extends Omit<ConnectionOptions, 'id' | 'g
   waitQueueTimeoutMS: number;
   /** If we are in load balancer mode. */
   loadBalanced: boolean;
+  /** @experimental */
+  minPoolSizeCheckIntervalMS: number;
```

Comment:
I did not want to reference a background thread in the name, so `minPoolSizeCheckIntervalMS` is what I came up with, open to other ideas though

Reply:
Name is clear to me, and it is internal if it bothers us later we could change it